### ISSUE TITLE
Move examples in `examples/` to docs and other improvements

### ISSUE THREAD

**Issue by aCampello** [wellcometrust/WellcomeML#79]
Would be good to have all examples in the docs webpage. I created a list to track

- [x] Bert Classifier
- [x] Bert embeddings
- [ ] BILSTM classifier
- [ ] CNN Classifier
- [ ] Entity Link
- [ ] Policy docs from s3
- [ ] Spacy Classifier
- [ ] Train NER Spacy

**Comment by lizgzil** [comment]
@aCampello think this can be closed?

**Comment by lizgzil** [comment]
oh ignore that!

**Comment by aCampello** [comment]
No, I think there are a lot of examples that haven't been moved to docs yet.
